Makefile

```diff
@@ -0,0 +1,191 @@
+# executables
+docker := $(shell which docker) --log-level error
+gcloud := $(shell which gcloud) --quiet --verbosity=error
+git := $(shell which git)
+go := $(shell which go)
+kubectl := $(shell which kubectl)
+
+# plugin's binary options
+binary_name = drone-gke
+
+# git options
+git_default_branch = $(shell cat $(CURDIR)/.git/refs/remotes/origin/HEAD | cut -d ' ' -f 2 | cut -d '/' -f 4)
+git_current_branch = $(shell $(git) rev-parse --abbrev-ref HEAD)
+git_current_revision = $(shell $(git) rev-parse --short HEAD)
+
+# plugin's docker options
+docker_default_repo_name = nytimes
+docker_image_name = drone-gke
+docker_default_tag = latest
+ifneq ($(git_current_branch), $(git_default_branch))
```

Comment:
add comment explaining that this is effectively:

if current branch name is NOT `master`
then use the current branch name as the default docker tag